upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -3836,6 +3863,20 @@ spec:
               name: kubernetes-services-endpoint
```

Comment:
You could use the `kubernetes-services-endpoint` config map that we now have for KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT.  All the components are alrady configured to pick that up.

Reply:
Wouldn't this require first to move to the Operator first?

Reply:
Not sure you need to do this conditionally. For cilium I chose to just always do this.

Reply:
Because we merge logic from different manifests, it will be easier later to track why these were added.

Reply:
> Wouldn't this require first to move to the Operator first?

No, we have the same config map in both install methods (but for operator it needs to be in the operator's namespace, not kube-system).  If you look a few lines above you can see the config map ref in your manifest ;-)

Reply:
Got it. Thanks! :)